infra/mock_api/template/ecs.tf

```diff
@@ -109,21 +106,132 @@ resource "aws_ecs_task_definition" "mock-api-ecs-task-definition" {
         {
           containerPort : 8080
         }
-      ],
+      ]
+      readonlyRootFilesystem = true
+      linuxParameters = {
+        capabilities = {
+          drop = ["ALL"]
+        },
+        initProcessEnabled = true
+      }
+      environment : [
+        {
+          "name" : "POSTGRES_USER"
+          "value" : "${data.aws_ssm_parameter.db_username.value}"
+        },
+        {
+          "name" : "POSTGRES_PASSWORD"
+          "value" : "${data.aws_ssm_parameter.db_pw.value}"
+        },
+        {
+          "name" : "POSTGRES_DB"
+          "value" : "${data.aws_ssm_parameter.db.value}"
+        },
+        {
+          "name" : "DB_HOST"
+          "value" : "test-wic-mt.cfsgsglh2mrn.us-east-1.rds.amazonaws.com"
+        }
+      ]
```

Comment:
Should this be something defined elsewhere and pulled in by a reference (if that's the right terminology in TF)?

Reply:
yep! This value is exported by the rds `resource` attribute, but hardcoded for testing purposes